config.yaml

```diff
@@ -303,6 +303,10 @@ options:
     default: false
     type: boolean
     description: Enable timescaledb extension
+  plugin_audit_enable:
+    default: false
```

Comment:
The same as VM, check default with Mohamed, please.

Reply:
Thanks for pointing that out, Alex! After syncing with Mohamed, I updated the plugin to enable it by default on https://github.com/canonical/postgresql-k8s-operator/pull/688/commits/32437181df919b938509ed6e6081ee54e2f1f59c.